packages/gatsby-legacy-polyfills/package.json

```diff
@@ -24,15 +24,15 @@
     "watch:polyfills": "microbundle -f iife -i src/polyfills.js --no-sourcemap --external=none --watch"
   },
   "dependencies": {
-    "core-js-compat": "^3.6.5"
+    "core-js-compat": "3.9.0"
```

Comment:
I'll pin these deps as we don't really need auto updates for these as they could break snapshots which is not ideal. Normally renovatebot should create prs to upgrade.